windows/src/developer/TIKE/xml/layoutbuilder/builder.js

```diff
@@ -350,6 +350,15 @@ $(function () {
     this.modifierCodes.SHIFT | this.modifierCodes.CTRL | this.modifierCodes.ALT
   ];
   
+  this.minimalModifierCombinations = [
+    0,
```

Comment:
Is this a placeholder for something?

Reply:
No. It's just no modifiers selected, so a bitmask of zero.